### PR TITLE
Motion: Replace deprecated APIs

### PR DESCRIPTION
**Pull request by yaraki**
Change-Id: Ia2aefc3e2aa5ba57222a3424b6a94cca8c1756ca

### CODE REVIEW

File: Motion/app/build.gradle [hunk 1]
```diff
@@ -19,12 +19,12 @@ apply plugin: 'kotlin-android'
 apply plugin: "androidx.navigation.safeargs.kotlin"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
 
     defaultConfig {
         applicationId 'com.example.android.motion'
         minSdkVersion 14
```
Comment (thagikura) [inline Motion/app/build.gradle:26]: minSdkVersion is deprecated https://developer.android.com/reference/tools/gradle-api/7.0/com/android/build/api/dsl/BaseFlavor#minSdkVersion(kotlin.Int)

Maybe change this to minSdk ?

Same thing applies for compileSdkVersion and targetSdkVersion

Comment (yaraki) [reply Motion/app/build.gradle]: Done

**Review by thagikura — APPROVED** [review]
Mostly LGTM with suggestions about the deprecated AGP DSL fields.

File: Motion/app/build.gradle [hunk 2]
```diff
@@ -19,12 +19,12 @@ apply plugin: 'kotlin-android'
 apply plugin: "androidx.navigation.safeargs.kotlin"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```
[the remark this answers is not in the document]
Comment (yaraki) [reply Motion/app/build.gradle]: Done

File: Motion/app/build.gradle [hunk 3]
```diff
@@ -19,12 +19,12 @@ apply plugin: 'kotlin-android'
...
 
     defaultConfig {
         applicationId 'com.example.android.motion'
         minSdkVersion 14
-        targetSdkVersion 30
+        targetSdkVersion 31
```
[the remark this answers is not in the document]
Comment (yaraki) [reply Motion/app/build.gradle]: Done

**Pull request merged**